Source/iOS/UIImage+Hue.swift

```diff
@@ -42,10 +42,11 @@ extension UIImage {
     let whiteColor = UIColor(red: 1, green: 1, blue: 1, alpha: 1)
     let colorSpace = CGColorSpaceCreateDeviceRGB()
     let raw = malloc(bytesPerRow * height)
-    let bitmapInfo = CGImageAlphaInfo.PremultipliedFirst.rawValue
-    let context = CGBitmapContextCreate(raw, width, height, bitsPerComponent, bytesPerRow, colorSpace, bitmapInfo)
-    CGContextDrawImage(context, CGRectMake(0, 0, CGFloat(width), CGFloat(height)), cgImage)
-    let data = UnsafePointer<UInt8>(CGBitmapContextGetData(context))
+    let bitmapInfo = CGImageAlphaInfo.premultipliedFirst.rawValue
+    let context = CGContext(data: raw, width: width, height: height, bitsPerComponent: bitsPerComponent, bytesPerRow: bytesPerRow, space: colorSpace, bitmapInfo: bitmapInfo)
+    context?.draw(cgImage, in: CGRect(x: 0, y: 0, width: CGFloat(width), height: CGFloat(height)))
+//    let data = UnsafePointer<UInt8>(context?.data)
```

Comment:
Yeah, I kept it for the sake of the argument, will remove the commented out line too. 😉